react-frontend/src/components/Nav/breadcrumbs.js

```diff
@@ -0,0 +1,47 @@
+import React, { useState, useEffect } from 'react';
+import { Link, useLocation, useHistory } from 'react-router-dom';
+import '../../css/breadcrumbs.css';
+
+const routeDisplayNames = {
```

Comment:
Since routes are already used in the Parent Router component. I'd opt to abstract the routes into a constant and then import them here.

Reply:
This const is exported from the routes.js file now